projects/maldoca/project.yaml

```diff
@@ -0,0 +1,13 @@
+
+homepage: "https://github.com/google/maldoca"
+language: c++
+primary_contact: "bgrill@google.com"
+auto_ccs:
+  - "dtao@google.com"
+  - "anise@google.com"
+sanitizers:
```

Comment:
Since there are no fuzzers or build instructions, are there any reason to specify which sanitizers to use? I would leave this one out unless you have specific conditions in mind.

Reply:
Thanks for pointing that out. Will remove for now.